modules/related-posts/related-posts.js

```diff
@@ -194,6 +194,13 @@
 		}
 	};
 
+	function afterPostsHaveLoaded() {
+		jprp.setVisualExcerptHeights();
+		$( '#jp-relatedposts a.jp-relatedposts-post-a' ).click(function() {
```

Comment:
I left these as they were, because I only moved them. But yeah, since I'm moving them, it doesn't hurt to fix the linting errors. Done in https://github.com/Automattic/jetpack/pull/10132/commits/14cee83bcc4a545331151fedd02176afdbc78b27.